public_website/tests/test_index.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+from django.urls import resolve, reverse
+from public_website import views
+
+class TestIndex(TestCase):
+
+    def test_index_url_calls_right_view(self):
+        resolved = resolve(reverse('index'))
+        self.assertEqual(resolved.func, views.index_view)
+
+    def test_index_view_calls_right_template(self):
+        response = self.client.get(reverse('index'))
```

Comment:
Tu peux aussi mettre 
```suggestion
        response = self.client.get("/")
```